{'description': {'en': 'Gentoo cross-compilation overlay for embedded devices'},
 'feed': ['https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/proj/embedded-cross.git/',
 'name': 'embedded-cross',
 'owner': [{'email': 'armin76@gentoo.org',
            'name': u'Ra\xfal Porcel',
            'type': 'project'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/proj/embedded-cross.git'}],
 'status': 'official'}
$ pmaint sync embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/eigenlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/eigenlay', named 'eigenlay', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
*** syncing embedded-cross
*** failed syncing embedded-cross
 * Sync failed with 1
 * Will try to re-create
$ pmaint sync embedded-cross
WARNING:pkgcore:repository at location '/home/mgorny/repos/pd-overlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/pd-overlay', named 'pd-overlay', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/pd-overlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/pd-overlay', named 'pd-overlay', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/eatnumber1' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/eatnumber1', named 'eatnumber1', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
Cloning into '/home/mgorny/repos/embedded-cross'...
*** syncing embedded-cross
*** synced embedded-cross
 * Sync succeeded after re-adding
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.063 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.061 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.062 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: app-editors/vim-core-7.2.303 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: app-editors/vim-core-7.3 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: x11-libs/gtk+-2.14.5 failed.
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '187' 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: app-editors/vim-core-7.0.235 failed.
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '176' 'depend'

!!! ERROR: app-editors/vim-core-7.0.243 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!        vim-core-7.2.303.ebuild, line   6:  Called inherit 'vim'
!!!                 ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'

!!! ERROR: app-editors/vim-core-7.1.123 failed.
!!!            vim-core-7.3.ebuild, line   7:  Called inherit 'vim'
!!!                 ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: app-editors/vim-core-7.2.442 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                 ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '248' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'

!!! ERROR: app-editors/vim-core-7.1.330 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'

!!! ERROR: app-editors/vim-7.3 failed.
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                 ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!                         ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '228' 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!                         ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'

!!! ERROR: app-misc/mdbus-9999 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.0.235.ebuild, line   5:  Called inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '160' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.0.243.ebuild, line   5:  Called inherit 'vim'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '165' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!        vim-core-7.1.123.ebuild, line   5:  Called inherit 'vim'

!!! ERROR: sci-geosciences/gpsd-2.37 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!        vim-core-7.2.442.ebuild, line   6:  Called inherit 'vim'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                 vim-7.3.ebuild, line   7:  Called inherit 'vim'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 173' for eclass vim

received unknown com: export CATEGORY='dev-lang' PR=r5 PVR='5.1.4-r5' PV='5.1.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/lua/lua-5.1.4-r5.ebuild' P='lua-5.1.4' PF='lua-5.1.4-r5' PN=luafailed
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe


!!! ERROR: x11-wm/enlightenment-0.16.999.063 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.1 failed.
!!!        vim-core-7.1.330.ebuild, line   5:  Called inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 188' for eclass vim

received unknown com: export CATEGORY='dev-lang' PR=r1 PVR='2.6.4-r1' PV='2.6.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.4-r1.ebuild' P='python-2.6.4' PF='python-2.6.4-r1' PN=pythonfailed
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                      ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!                  ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '178' 'depend'
!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                  ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!                  ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '245' 'depend'

!!! ERROR: x11-base/xglamo-20081205-r2 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                      ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.2.303
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.235
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.243
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.123
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.2.442
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi '0' doesn't match sourced eapi '2' while processing ebuild src: sci-geosciences/fso-gpsd-0.8
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-lang/lua-5.1.4-r5
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.330
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-lang/python-2.6.4-r1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-geosciences/gpsd-2.37
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/pkgconfig-0.23
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.6
!!! caught exception Metadata Exception: pkg ebuild src: x11-wm/enlightenment-0.16.999.063, attr eapi
error: parsed eapi '2' doesn't match sourced eapi '0' while processing ebuild src: x11-wm/enlightenment-0.16.999.063
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/perl-5.8.8-r9
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-20091021
!!! caught exception Metadata Exception: pkg ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1, attr eapi
error: parsed eapi '0' doesn't match sourced eapi '3' while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/strace-4.5.18
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/python-2.5.4-r3
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-base/xglamo-20081205-r2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework
!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.5 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: sys-mobilephone/fso-specs-2010.05.11.2 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore

!!! Call stack:
!!! ERROR: dev-lang/python-2.5.4-r3 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild'
!!!                      ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '285' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!!                      ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/apm2-2.0.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: net-wireless/wmiconfig-9999 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '201' 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                          ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                      ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                      ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild'
!!!              ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/fsogsmd-0.5.0 failed.
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!!                          ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '218' 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'
!!!              isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5 failed.
!!!              ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!   enlightenment-0.16.999.063.ebuild, line   8:  Called inherit 'enlightenment'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                      ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '285' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '188' 'depend'
!!!               gpsd-2.37.ebuild, line   5:  Called inherit 'eutils' 'distutils' 'autotools'
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'

!!! ERROR: sys-mobilephone/libframeworkd-glib-0.1 failed.
!!!              ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '245' 'depend'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '183' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 296' for eclass enlightenment

!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 245' for eclass distutils

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090704 PV=20090704 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild' P='freesmartphone-framework-20090704' PF='freesmartphone-framework-20090704' PN='freesmartphone-framework'path
received unknown com: export CATEGORY='x11-wm' PR=r0 PVR='0.16.999.062' PV='0.16.999.062' EBUILD='/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.062.ebuild' P='enlightenment-0.16.999.062' PF='enlightenment-0.16.999.062' PN=enlightenmentpath
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '193' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.5.4-r3.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                      ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                    ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!              ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!              ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '198' 'depend'
!!!                    ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.5.4-r3.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.5.4-r3.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                    ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '275' 'depend'
!!!   freesmartphone-framework-0.5.1.ebuild, line   5:  Called inherit 'distutils'
!!!              ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!      xglamo-20081205-r2.ebuild, line   5:  Called inherit 'autotools' 'git' 'toolchain-funcs'
!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild'
!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!                    ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '245' 'depend'
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!         python-2.5.4-r3.ebuild, line  12:  Called inherit 'autotools' 'eutils' 'flag-o-matic' 'libtool' 'multilib' 'pax-utils' 'python' 'toolchain-funcs' 'versionator'
!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                       ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 210' for eclass distutils

!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.1' PV='0.9.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild' P='fso-abyss-0.9.1' PF='fso-abyss-0.9.1' PN='fso-abyss'path
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'pax-utils'

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!!              ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!              apm2-2.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!   freesmartphone-framework-0.5.5.ebuild, line   6:  Called inherit 'git'
!!!                            ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!   fso-specs-2010.05.11.2.ebuild, line   6:  Called inherit 'base' 'autotools' 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 203' for eclass pax-utils

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild' P='fsodatad-0.1.0' PF='fsodatad-0.1.0' PN=fsodatadpath
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild'
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'

!!! ERROR: sys-mobilephone/fsomusicd-0.0.0 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                            ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!           fsogsmd-0.5.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!          wmiconfig-9999.ebuild, line   5:  Called inherit 'subversion' 'toolchain-funcs'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 218' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild' P='fsonetworkd-0.1.0' PF='fsonetworkd-0.1.0' PN=fsonetworkdpath
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe


!!! ERROR: sys-mobilephone/fsousaged-0.9.4 failed.
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '198' 'depend'

!!! ERROR: sys-mobilephone/fsoinitd-0.1.0 failed.
!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'subversion'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '213' 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   freesmartphone-framework-0.5.ebuild, line   6:  Called inherit 'git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 228' for eclass subversion

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.3.1' PV='0.9.3.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/gsm0710muxd/gsm0710muxd-0.9.3.1.ebuild' P='gsm0710muxd-0.9.3.1' PF='gsm0710muxd-0.9.3.1' PN=gsm0710muxdpath
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   libframeworkd-glib-0.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!          ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                     ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'
-0.5.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
!!! caught exception Metadata Exception: pkg ebuild src: sys-mobilephone/fso-specs-2010.05.11.2, attr eapi
error: parsed eapi '2' doesn't match sourced eapi '0' while processing ebuild src: sys-mobilephone/fso-specs-2010.05.11.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/apm2-2.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-wireless/wmiconfig-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsogsmd-0.5.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5
!!! caught exception Metadata Exception: pkg ebuild src: sys-mobilephone/libframeworkd-glib-0.1, attr eapi
error: parsed eapi '0' doesn't match sourced eapi '2' while processing ebuild src: sys-mobilephone/libframeworkd-glib-0.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-wm/enlightenment-0.16.999.062
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-abyss-0.9.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodatad-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfso-glib-2010.05.11.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsomusicd-0.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsonetworkd-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsotransport-0.9.6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/gsm0710muxd-0.9.3.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-ui-shr-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libphone-uti
!!! ERROR: sys-mobilephone/libframeworkd-glib-20090614 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 266' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild' P='libframeworkd-glib-99999999' PF='libframeworkd-glib-99999999' PN='libframeworkd-glib'failed
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '203' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!                   ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                   ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/libframeworkd-glib-20090227 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild'
!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-99999999.ebuild' P='libframeworkd-phonegui-99999999' PF='libframeworkd-phonegui-99999999' PN='libframeworkd-phonegui'path
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild'
!!!                   ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '266' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090724 PV=20090724 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild' P='libframeworkd-phonegui-20090724' PF='libframeworkd-phonegui-20090724' PN='libframeworkd-phonegui'path
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!                   ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         fsomusicd-0.0.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!                   ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!          fsoinitd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 238' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.8.1' PV='0.8.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild' P='libfsoframework-0.8.1' PF='libfsoframework-0.8.1' PN=libfsoframeworkpath
!!!         fsousaged-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                   ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '266' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 228' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.2' PV='0.9.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710mux/libgsm0710mux-0.9.2.ebuild' P='libgsm0710mux-0.9.2' PF='libgsm0710mux-0.9.2' PN=libgsm0710muxpath
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!                   ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 233' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.1' PV='0.1.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoresource/libfsoresource-0.1.1.ebuild' P='libfsoresource-0.1.1' PF='libfsoresource-0.1.1' PN=libfsoresourcepath
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!   libframeworkd-glib-20090614.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 246' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090724 PV=20090724 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-utils/libphone-utils-20090724.ebuild' P='libphone-utils-20090724' PF='libphone-utils-20090724' PN='libphone-utils'failed
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   libframeworkd-glib-20090227.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.0.0' PV='1.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mkdump/mkdump-1.0.0.ebuild' P='mkdump-1.0.0' PF='mkdump-1.0.0' PN=mkdumpfailed
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

ls-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090506
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mdbus2-2.0.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/mkdump-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090724
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsoframework-0.8.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phonefsod-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libgsm0710mux-0.9.2
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneuid-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsoresource-0.1.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/pyphonelog-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-utils-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/shr-contacts-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mterm2-1.9.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/shr-contacts-20090724
Error running an operation:
 Unhandled exception in regen_cache
 * Cache regen failed with 246
